packages/app-store/slackmessaging/api/callback.ts

```diff
@@ -52,7 +53,7 @@ async function handler(req: NextApiRequest, res: NextApiResponse) {
     },
   });
 
-  res.redirect("/apps/installed");
+  res.redirect(getInstalledAppPath({ variant: "conferencing", slug: "slack" }));
```

Comment:
This is to support redirecting to the correct category within the Installed Apps page once the app has been installed.